x/ecocredit/simulation/genesis.go

```diff
@@ -165,92 +191,185 @@ func RandomizedGenState(simState *module.SimulationState) {
 	simState.GenState[ecocredit.ModuleName] = bz
 }
 
+func createClass(ctx context.Context, sStore api.StateStore, class *api.Class) (uint64, error) {
+	cKey, err := sStore.ClassTable().InsertReturningID(ctx, class)
+	if err != nil {
+		return 0, err
+	}
+
+	seq, err := sStore.ClassSequenceTable().Get(ctx, class.CreditTypeAbbrev)
+	if err != nil {
+		if ormerrors.IsNotFound(err) {
+			if err := sStore.ClassSequenceTable().Save(ctx, &api.ClassSequence{
```

Comment:
We are not expecting a sequence table to exist if the credit class does not exist, in which case we should use `Insert` rather than `Save`. `Insert` will error if the sequence table already exists, which should not be the case here and would be good to test to make sure.

```suggestion
			if err := sStore.ClassSequenceTable().Insert(ctx, &api.ClassSequence{
```